cli/utils/index.ts

```diff
@@ -43,6 +43,13 @@ export const assertShellResult = (result: ShellString, errMsg: string) => {
   }
 }
 
+export const assertFindings = (findings: Finding[]) => {
+  const byteLength = Buffer.byteLength(JSON.stringify(findings));
+
+  if(byteLength > 50000) throw Error(`Cannot return more than 50kB of findings per request (received ${byteLength} bytes)`)
```

Comment:
we might need to adjust this since 1kb = 1,024 bytes

Reply:
Updated to use correct kb size